conf/application.conf

```diff
@@ -50,29 +54,158 @@ play.i18n.langs = ["en"]
+play.application.loader = "uk.gov.hmrc.play.bootstrap.ApplicationLoader"
+
+# Primary entry point for all HTTP requests on Play applications
+play.http.requestHandler = "routing.VersionRoutingRequestHandler"
+
+# Provides an implementation of AuditConnector. Use `uk.gov.hmrc.play.bootstrap.AuditModule` or create your own.
+# An audit connector must be provided.
+play.modules.enabled += "uk.gov.hmrc.play.audit.AuditModule"
+
+# Provides an implementation of MetricsFilter. Use `uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule` or create your own.
+# A metric filter must be provided
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
+
+# Provides an implementation and configures all filters required by a Platform backend microservice.
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.backend.BackendModule"
+
+play.filters.enabled += "play.filters.cors.CORSFilter"
+
+play.filters.cors { allowedOrigins = ["http://localhost:9680"] }
+
+# Json error handler
+play.http.errorHandler = "utils.ErrorHandler"
+
+# Play Modules
+# ~~~~
+# Additional play modules can be added here
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
+play.modules.enabled += "config.SavingsIncomeModule"
+
+# Evolutions
 # ~~~~~
-# By default all controllers will have authorisation, logging and
-# auditing (transaction monitoring) enabled.
-# The below controllers are the default exceptions to this rule.
-
-controllers {
-  uk.gov.hmrc.individualssavingsincomeapi.controllers.MicroserviceHelloWorldController = {
-    needsAuth = false
-    needsLogging = false
-    needsAuditing = false
+# You can disable evolutions if needed
+# evolutionplugin=disabled
+
+# Api related config
+api {
+  # The status of the version of the API for the API Platform.
+  1.0 {
+    status = "BETA"
+    endpoints.enabled = true
+  }
+
+  confidence-level-check {
+    confidence-level = 250
+    definition.enabled = true
+    auth-validation.enabled = true
   }
+
+  # The context which the API will have via the API Platform http://API_GATEWAY/{api.gateway.context}/
+  gateway.context = "individuals/savings-income"
+}
+
+bootstrap.http.headersAllowlist = [ "Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id" ]
+
+internalServiceHostPatterns = [ "localhost" ]
+
+# Metrics plugin settings - graphite reporting is configured on a per env basis
+metrics {
+  name = ${appName}
+  rateUnit = SECONDS
+  durationUnit = SECONDS
+  showSamples = true
+  jvm = true
+  enabled = true
 }
 
 # Microservice specific config
 
+auditing {
+  enabled = true
+  consumer {
+    baseUri {
+      host = localhost
+      port = 8100
+    }
+  }
+}
+
+feature-switch {
+  opw.enabled = true
+  allowTemporalValidationSuspension.enabled = true
+  postCessationReceipts.enabled = "true"
+  passDeleteIntentHeader.enabled = true
```

Comment:
Certainly opw and postCessationReceipts receipts aren't used and can be removed from AppConfig and here. Not so worried about the other two as they're more general.